### PR TITLE
phan: Stub more constants

### PR DESCRIPTION
**Pull request by anomiex**
## Proposed changes:

* More basic WordPress constants that don't seem to have any other way to get them, like `WPINC` and `WP_CONTENT_DIR`.
* Add cookie-related constants, seems WordPress doesn't have any generic setcookie wrapper after all.
* crm: Add a stub for constants defined by `ZeroBSCRM::define()`, as Phan doesn't recognize that as defining constants.

### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?
- [ ] Have you tested your changes on WordPress.com, if applicable (if so, you'll see a generated comment below with a script to run)?

## Jetpack product discussion

None

## Does this pull request change what data or activity we track or use?

No

## Testing instructions:

* Phan happy?

### CODE REVIEW

File: .phan/stubs/wordpress-constants.php [hunk 1]
```diff
@@ -24,11 +23,15 @@
...
 define( 'WP_DEBUG', (bool) $v );
 define( 'WP_DEBUG_LOG', (bool) $v );
-define( 'WP_PLUGIN_DIR', './' );
-define( 'WPMU_PLUGIN_DIR', './' );
-define( 'EMPTY_TRASH_DAYS', 30 * 86400 );
 define( 'SCRIPT_DEBUG', (bool) $v );
-define( 'WP_LANG_DIR', './' );
+define( 'WPINC', 'wp-includes' );
+define( 'WP_CONTENT_DIR', ABSPATH . 'wp-content' );
+define( 'WP_PLUGIN_DIR', WP_CONTENT_DIR . '/plugins' );
```
Comment (tbradsha) [inline .phan/stubs/wordpress-constants.php]: Noting that these have [long been discouraged](https://codex.wordpress.org/Determining_Plugin_and_Content_Directories) for use, though perhaps that's more of a PHPCS-type check (and as noted here, there's not a good alternative function).

File: .phan/stubs/wordpress-constants.php [hunk 2]
```diff
@@ -24,11 +23,15 @@
...
+define( 'WPMU_PLUGIN_DIR', WP_CONTENT_DIR . '/mu-plugins' );
+define( 'WP_LANG_DIR', WP_CONTENT_DIR . '/languages' );
+define( 'WP_CONTENT_URL', 'http://example.com/wp-content' );
+define( 'WP_PLUGIN_URL', WP_CONTENT_URL . '/plugins' );
+define( 'WPMU_PLUGIN_URL', WP_CONTENT_URL . '/mu-plugins' );
```
Comment (tbradsha) [inline .phan/stubs/wordpress-constants.php]: Typically people should use [`plugins_url()`](https://developer.wordpress.org/reference/functions/plugins_url/) instead.

File: .phan/stubs/wordpress-constants.php [hunk 3]
```diff
@@ -80,3 +104,6 @@
 
 // Templating-related WordPress constants.
 define( 'WP_DEFAULT_THEME', 'twentytwentywhenever' );
+
+// Testing constants.
+define( 'WP_TESTS_DOMAIN', 'example.org' );
```
Comment (tbradsha) [inline .phan/stubs/wordpress-constants.php:112]: Might be worth mentioning that this isn't a constant in WP, but is used in PHPUnit tests.

File: .phan/stubs/wordpress-constants.php (hunk 1 above)
Comment (anomiex) [reply .phan/stubs/wordpress-constants.php]: Thanks, I was looking for information like that link and wasn't able to find it anywhere.

**Review by tbradsha — APPROVED** [review]
As of this point, looks good.

**Pull request merged**